web-app/pages/index.tsx

```diff
@@ -18,16 +18,30 @@ const Home: NextPage = () => {
 
   return (
     <Layout>
-      <h1 className={styles.title}>Welcome to NEAR University&rsquo;s Certificate Browser!</h1>
-
-      <p className={styles.description}>Choose an account to view:</p>
-
-      <form onSubmit={onSubmit} className="mt-4">
-        <input type="text" name="account" placeholder={`example${suffix}`} onChange={(event) => setAccount(event.target.value)} className="border border-gray-500 p-2" required />
-        <button type="submit" className="border border-gray-500 p-2">
-          Go
-        </button>
-      </form>
+      <div className="">
```

Comment:
All of these changes here in this diff in index.tsx seem like the same problem @ozanisgor is having. It seems like you haven't rebased with the develop branch. These changes already happened in develop. So if *you* didn't work on index.tsx for your feature, there should be no diff (changes) shown here.

Reply:
I think all of us need a live tutorial about how to do a clear rebase from you.

cc @ozanisgor @rashaabdulrazzak